### ISSUE TITLE
$jsonexport deprecation

### ISSUE THREAD

**Issue by SirbitoX** [Elgg/Elgg#8256]
Hello,
As Elgg says:
>Using $jsonexport to produce json output has been deprecated

but in "web_services" plugin and in "\views\json\api\output.php" file we are using this global parameter:
```php
$result = $vars['result'];
$export = $result->export();

global $jsonexport;

// with api calls, we don't want extra baggage found in other json views
// so we skip the associative array
$jsonexport = $export;
```
and in every last version of Elgg this plugin is attached as default. 
Do you have any plan to replace last line of the code with ```echo json_encode($export);```???

Thanks in advance.

**Comment by brettp** [comment]
Related to #8053.

**Comment by SirbitoX** [comment]
@brettp I didn't get the relation!

**Comment by jdalsem** [comment]
Fixed with #8572

**Comment by SirbitoX** [comment]
:+1:
